### ISSUE TITLE
Turning and rotating 3D Tshirt

### ISSUE THREAD

**Issue by Arakistech** [adrianhajdin/project_threejs_ai#4]
Hi, I would like to rotate the Tshirt as it is a .GLB file with Three.js but that is not currently possible. How can I do it?

**Comment by Raspber** [comment]
i am learning how to upload my own glb files, that also requires some knowledge of blender so i can have it upload to certain parts of the model. im trying to learn three js for this. this way i can add more products and view options as well as locations on the product.

**Comment by pradeepth02** [comment]
As soon as i click on ai full or ai logo button my screen goes white and i can see errors on the server terminal and even in inspect console.
![Screenshot (8)](https://github.com/adrianhajdin/project_threejs_ai/assets/115602871/3f35cfde-1f1a-4d50-9e24-91f7f745ab1e)
![Screenshot (9)](https://github.com/adrianhajdin/project_threejs_ai/assets/115602871/712d6f9b-014d-4565-a1fd
![Screenshot (10)](https://github.com/adrianhajdin/project_threejs_ai/assets/115602871/ce78e289-260d-481d-b3f4-38cfe98ab756)
-8d91beeafb68)
![Screenshot (11)](https://github.com/adrianhajdin/project_threejs_ai/assets/115602871/e8dba14d-2033-4c15-85ae-04699646b1ea)
Can someone please help me out with this....?

**Comment by K4R7IK** [comment]
@pradeepth02 Hey, Just want to tell you that if you face an issue, no need to post it everywhere as it's get irritating. You can create an issue or post somewhere where your issue is related with others. Posting your problem everywhere on GitHub isn't a good practice & is unprofessional.

**Comment by Raspber** [comment]
> @Arakistech @Raspber I also wanted to rotate the 3d model. I used *OrbitalControl* from *react/drei*. The issue I'm facing is that, there is some kind of wall behind the T-shirt which is preventing it from rotating.

Yes I got it to rotate. You need to go into the blender file and delete the wall. You also need to add 2 more lights and delete the shadow that is baked in.

**Comment by K4R7IK** [comment]
> > @Arakistech @Raspber I also wanted to rotate the 3d model. I used _OrbitalControl_ from _react/drei_. The issue I'm facing is that, there is some kind of wall behind the T-shirt which is preventing it from rotating.
> 
> Yes I got it to rotate. You need to go into the blender file and delete the wall. You also need to add 2 more lights and delete the shadow that is baked in.

Can you share the blender file with me? Never used blender before. Therefore, it would be great.
